docs/flutter/v2/guides/quickstart.mdx

```diff
@@ -5,316 +5,868 @@ nav: 2.0
+  @override
+  void onJoin({required HMSRoom room}) {
+    room.peers?.forEach((peer) {
+      if (peer.isLocal) {
+        localPeer = peer;
+        if (peer.videoTrack != null) {
+          localPeerVideoTrack = peer.videoTrack;
+        }
+        if (mounted) {
+          setState(() {});
+        }
+      }
+    });
+  }
+
+  // Called when there's a peer update - use to update local & remote peer variables
+  @override
+  void onPeerUpdate({required HMSPeer peer, required HMSPeerUpdate update}) {
+    switch (update) {
+      case HMSPeerUpdate.peerJoined:
+        if (!peer.isLocal) {
+          if (mounted) {
+            setState(() {
+              remotePeer = peer;
+            });
+          }
+        }
+        break;
+      case HMSPeerUpdate.peerLeft:
+        if (!peer.isLocal) {
+          if (mounted) {
+            setState(() {
+              remotePeer = null;
+            });
+          }
+        }
+        break;
+      case HMSPeerUpdate.networkQualityUpdated:
+        return;
+      default:
+        if (mounted) {
+          setState(() {
+            localPeer = null;
+          });
+        }
+    }
+  }
+```
 
-/// video off
-hmsSDK.switchVideo(isOn: false);
+### Step 5: Listen to track updates
+
+100ms SDK also sends updates to the application about any change in `HMSTrack` via the callbacks in `HMSUpdateListener`. Our application must listen to the corresponding updates in `onTrackUpdate`. Check the [Update Listeners documentation](./../features/update-listener-enums#hms-track-update) to understand the types of updates emitted by the SDK for track updates.
+
+```dart
+// Called when there's a track update - use to update local & remote track variables
+  @override
+  void onTrackUpdate(
+      {required HMSTrack track,
+      required HMSTrackUpdate trackUpdate,
+      required HMSPeer peer}) {
+    if (track.kind == HMSTrackKind.kHMSTrackKindVideo) {
+      switch (trackUpdate) {
+        case HMSTrackUpdate.trackRemoved:
+          if (mounted) {
+            setState(() {
+              peer.isLocal
+                  ? localPeerVideoTrack = null
+                  : remotePeerVideoTrack = null;
+            });
+          }
+          return;
+        default:
+          if (mounted) {
+            setState(() {
+              peer.isLocal
+                  ? localPeerVideoTrack = track as HMSVideoTrack
+                  : remotePeerVideoTrack = track as HMSVideoTrack;
+            });
+          }
+      }
+    }
+  }
 ```
 
-### Leave a video call
+### Step 6: Other callbacks
+
+100ms SDK provides various other callbacks to handle different scenarios in the app. For example, you can use `onAudioDeviceChanged` to get updates whenever a new audio device or an audio device is switched. Please check [here](./../features/update-listeners) for more information about these callbacks. 
+
+```dart
+// More callbacks - no need to implement for quickstart
+  // More callbacks - no need to implement for quickstart
```

Comment:
`// More callbacks ... ` comment is repeated here

Reply:
done